File: Clase 11/Backend/Domain/Homework.cs
using System;
namespace Domain;

public class Homework
{
    public string Id { get; set; }
    public string Description { get; set; }
    public DateTime DueDate  { get; set; }
    public int Score { get; set; }
    public int Rating { get; set; }
    public List<Exercise> exercises { get; set; }
}

File: Clase 11/src/app/models/Homework.ts
import { Exercise } from './Exercise';

export class Homework {
    id: string;
    description: string;
    dueDate: Date;
    score: number;
    exercises: Array<Exercise>;
    rating: number;
    constructor(id:string, description:string, score:number, dueDate:Date, exercises:
Array<Exercise>, rating:number){
        this.id = id;
        this.description = description;
        this.score = score;
        this.dueDate = dueDate;
        this.exercises = exercises;
        this.rating = rating;
    }
}
